Ensure that there is sufficient contrast between the graded-group title and the background. (#336)
Khan/perseus: .changeset/olive-pillows-invent.md

---
"@khanacademy/perseus": patch
---

> Another microfix that updates a title to use the next darker level of gray so that it can pass our a11y requirements.

Reviewer: These changeset descriptions become part of our changelog.  I'd recommend a simple declarative statement of the change that will make sense in that context.  Maybe something like this: 
```suggestion
Update graded-group widget title to use a darker gray to meet a11y requirements
```

Feel free to tweak that further.
